### PR TITLE
IGAPP-576: Remove feedback type selection

### PR DESCRIPTION
**Pull request by Taggotty**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: web/src/modules/feedback/components/FeedbackBoxContainer.js [hunk 1]
```diff
@@ -1,55 +1,40 @@
...
 type PropsType = {|
-  cities: ?Array<CityModel>,
-  title?: string,
   alias?: string,
   path?: string,
-  query?: string,
   isPositiveRatingSelected: boolean,
   location: LocationState,
-  offers: ?Array<OfferModel>,
-  postFeedbackDataOverride?: FeedbackParamsType => void,
```
Comment (Taggotty) [inline web/src/modules/feedback/components/FeedbackBoxContainer.js:43]: removed unused properties.

File: web/src/modules/feedback/components/FeedbackBoxContainer.js [hunk 2]
```diff
@@ -60,189 +45,62 @@ type StateType = {|
...
-      feedbackType: selectedFeedbackOption.feedbackType,
-      feedbackCategory: selectedFeedbackOption.feedbackCategory,
+      feedbackType: feedbackType,
       isPositiveRating: isPositiveRatingSelected,
       comment,
       permalink: path,
       city: city || INTEGREAT_INSTANCE,
       language: language || DEFAULT_FEEDBACK_LANGUAGE,
-      alias: feedbackAlias,
-      query
```
Comment (klinzo) [inline web/src/modules/feedback/components/FeedbackBoxContainer.js:223]: Why did you remove this? Because we have no feedback for the search, right?

File: web/src/modules/feedback/components/FeedbackBox.js [hunk 3]
```diff
@@ -66,8 +58,6 @@ export const FeedbackBox = ({
 }: PropsType) => (
   <StyledFeedbackBox>
     <ModalHeader t={t} closeFeedbackModal={closeFeedbackModal} title={t('feedback')} />
-    <Description>{t('feedbackType')}</Description>
-    <Dropdown items={feedbackOptions} selectedItem={selectedFeedbackOption} onOptionChanged={onFeedbackOptionChanged} />
```
Comment (klinzo) [inline web/src/modules/feedback/components/FeedbackBox.js:70]: Do we still use this dropdown somewhere or could we remove it? Does it use a library which could be removed?

**Review by klinzo — APPROVED** [review]
Tests, tests, tests :tada: :t-rex:

File: web/src/modules/feedback/FeedbackVariant.js [hunk 4]
```diff
@@ -1,28 +1,16 @@
 // @flow
```
[the remark this answers is not in the document]
Comment (Taggotty) [reply web/src/modules/feedback/FeedbackVariant.js:1]: actually it is not used anymore. but in the dropdown component. I removed both the `Dropdown` component (as it was only used for feedback and is not used anymore) and the `FeedbackVariant`.

File: web/src/modules/feedback/components/FeedbackBox.js (hunk 3 above)
Comment (Taggotty) [reply web/src/modules/feedback/components/FeedbackBox.js:70]: Removed with unused libraries.

File: web/src/modules/feedback/components/FeedbackBoxContainer.js (hunk 2 above)
Comment (Taggotty) [reply web/src/modules/feedback/components/FeedbackBoxContainer.js:223]: `SearchFeedback.js` implements this property, but here it is not necessary.

**Review by SteffiStuffel — APPROVED** [review]
